userops_ext.go

```diff
@@ -55,6 +55,19 @@ func (u *UserOperationExt) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (u *UserOperationExt) MarshalJSON() ([]byte, error) {
```

Comment:
add test case too.

Reply:
please add a test case that include a userops which has UserOperationExt MarshalJSON() coverage